src/System.Private.CoreLib/shared/System/Resources/ManifestBasedResourceGroveler.cs

```diff
@@ -463,6 +463,33 @@ private void HandleSatelliteMissing()
             throw new MissingSatelliteAssemblyException(SR.Format(SR.MissingSatelliteAssembly_Culture_Name, _mediator.NeutralResourcesCulture, satAssemName), missingCultureName);
         }
 
+        private static string GetManifestResourceNamesList(Assembly assembly)
+        {
+            try
+            {
+                string [] manifestResourceNames = assembly.GetManifestResourceNames();
+                if (manifestResourceNames.Length == 0)
+                {
+                    return "[]";
+                }
+
+                StringBuilder sb = StringBuilderCache.Acquire();
+                sb.Append('[');
+                sb.Append(manifestResourceNames[0]);
+                for (int i = 1; i < manifestResourceNames.Length; i++)
+                {
+                    sb.Append(", ");
```

Comment:
Resource stream names can contain spaces (at least, CSC allows it). So I suggest quoting them. ie 
```
@"""" + String.Join(@""", """, manifestResourceNames) + @"""";
```

Reply:
I'll add the quotes.